pyscriptjs/src/utils.ts

```diff
@@ -111,3 +103,28 @@ export function getAttribute(el: Element, attr: string): string | null {
     }
     return null;
 }
+
+export function createAlertBanner(message: string, level: "error" | "warning" = "error", logMessage = true) {
+
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+    switch(`log-${level}-${logMessage}`) {
+        case "log-error-true":
+        console.error(message);
+        break;
+        case "log-warning-true":
+        console.warn(message)
+        break;
+    }
+
+    const banner = document.createElement("div")
+    banner.className = `alert-banner py-${level}`
+    banner.textContent = message
+
+    const closeButton = document.createElement("button")
+    closeButton.id = "alert-close-button"
+    closeButton.addEventListener("click", () => banner.remove() )
+    closeButton.innerHTML = `<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill="currentColor" width="12px"><path d='M.293.293a1 1 0 011.414 0L8 6.586 14.293.293a1 1 0 111.414 1.414L9.414 8l6.293 6.293a1 1 0 01-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 01-1.414-1.414L6.586 8 .293 1.707a1 1 0 010-1.414z'/></svg>`
```

Comment:
genuine question: is this considered a good way to inline svg elements?
I would expect at least it to be a constant so that it can be reused elsewhere, and/or put in its own file which is included by rollup?
But I don't really know what is the idiom in web development, so if this is the "right" way to do it, I'm fine with it

Reply:
Good question, you can do this kind of things in React when you want to create a SVG icon (kind of there are some other bits you need to do), but I'm not sure if this is bad in pure JS?   I've also seen places where they use an `img` tag and pass the svg code, something along the lines of `<img src="data:image/svg+xml;charset=utf-8,%3Csvg...">`

Although moving it to a const might be a good idea so we can reuse it elsewhere if needed!